oracles/template.env

```diff
@@ -10,7 +10,7 @@ CHAIN_ID=31337
 WEB3_RPC_URL="http://127.0.0.1:8545/"
 PRIVATE_KEY="0x"
 ORACLE_ADDRESS="0x"
-ORACLE_ABI_PATH="../arbitrum-contracts/artifacts/contracts/Oracle.sol/Oracle.json"
+ORACLE_ABI_PATH=""
```

Comment:
Could add a correct path then where the contracts get deployed to
`../contracts/artifacts/contracts/Oracle.sol/Oracle.json` I think

Reply:
Correct path is abi/ChatOracle.json, 
`ORACLE_ABI_PATH = os.getenv("ORACLE_ABI_PATH", "abi/ChatOracle.json")` in settings.py so it doesn't need value